src/app/header/header.component.html

```diff
@@ -1,33 +1,33 @@
-<nav class="navbar navbar-default navbar-pf" role="navigation">
-	<div class="navbar-header">
-		<button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse-1">
+<nav id="header_wrapper" class="navbar navbar-default navbar-pf" role="navigation">
+	<div id="header_nav" class="navbar-header">
+		<button type="button" id="header_navToggleBtn" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse-1">
 			<span class="sr-only">Toggle navigation</span>
 			<span class="icon-bar"></span>
-      		<span class="icon-bar"></span>
-      		<span class="icon-bar"></span>
-    	</button>
-    	<img src="../../assets/img/brand.png" alt="Almighty" />
+			<span class="icon-bar"></span>
+			<span class="icon-bar"></span>
+    </button>
+  	<img id="header_ALMLogo" src="../../assets/img/brand.png" alt="Almighty" />
   </div>
   <div class="collapse navbar-collapse navbar-collapse-1">
-	  <ul class="nav navbar-nav navbar-utility">
+	  <ul id="header_rightDropdown" class="nav navbar-nav navbar-utility">
       	<li class="dropdown">
-        	<a href="#" class="dropdown-toggle" data-toggle="dropdown">
+        	<a href="#" id="header_dropdownToggle" class="dropdown-toggle" data-toggle="dropdown">
           		<span class="pficon pficon-user"></span>
           		John <b class="caret"></b>
```

Comment:
We need to add span tags around John and then add an ID. We plan to display the logged in user's name.